packages/nimble-components/src/table/template.ts

```diff
@@ -22,7 +22,7 @@ export const template = html<Table>`
     <template role="table" ${children({ property: 'childItems', filter: elements() })}>
         <div class="table-container">
             <div role="rowgroup" class="header-container">
-                <div class="header-row" role="row">
+                <div class="header-row" role="row" style="grid-template-columns: ${x => x.rowGridColumns} auto; left: -${x => x.scrollX}px;">
```

Comment:
Just a note that we may still want to followup with Visual Design / Rick about the 'full height scrollbar' option. This branch/PR still has the scrollbar starting below the header row, which requires us to hook up to the `scroll` event and update the `left` offset of the header row programmatically (so it scrolls in sync with the table rows, if those require a horizontal scrollbar). Now that we aren't hardcoding scrollbar widths I don't think this option is that problematic though (but we can get reviewer feedback on that).

I took the other version (from Molly's prototype) and made a new branch off of this one with the full-height scrollbar, for comparison / in case we end up wanting to switch to that.
[Commit](https://github.com/ni/nimble/commit/98a1a23450e2781c8548089baf01348a434380d4)
[Storybook/Demo](https://60e89457a987cf003efc0a5b-vphzhwrlhh.chromatic.com/?path=/story/table--table&args=data:LargeDataSet)

Reply:
Checked in with @RickA-NI and he felt that keeping the scrollbar under the header was the right way to go.